# A fiducial to fixed frame transform of a fiducial marker
int32 fiducial_id
geometry_msgs/Transform transform